install-cockroachdb.md

```diff
@@ -193,7 +193,7 @@ $(document).ready(function(){
 
     <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="mac-source-step3"><span class="gp" data-eventcategory="mac-source-step3">$ </span><span class="nb">cd</span> <span class="nv">$GOPATH</span>/src/github.com/cockroachdb/cockroach<br>$ git checkout {{site.data.strings.version}}<br><span class="gp">$ </span>make install</code></pre></div>
 
-    <p>The first time you run <code class="highlighter-rouge">make install</code>, it can take awhile to download and install various dependencies.</p>
+    <p><div class="bs-callout bs-callout-info">The <code class="highlighter-rouge">make install</code> command builds a binary covered by the CockroachDB Community License. To build a pure open-source (APL2) version instead, use <code>make buildoss</code>. See this <a href="https://www.cockroachlabs.com/blog/how-were-building-a-business-to-last/">blog post</a> for more details. Also, note that the first time you run <code>make</code>, it can take awhile to download and install various dependencies.</div></p>
```

Comment:
Might mention here that the default binary is a hybrid license with core functionally covered by APL2 and enterprise functionality covered by the CCL.